ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/EthTokenMetadata.kt

```diff
@@ -0,0 +1,15 @@
+package org.p2p.ethereumkit.external.model
+
+import org.p2p.ethereumkit.internal.models.EthAddress
+import java.math.BigDecimal
+import java.math.BigInteger
+
+data class EthTokenMetadata(
+    val contractAddress: EthAddress,
+    val balance: BigInteger,
+    val decimals: Int,
```

Comment:
maybe you can create method for getting balance in lamports?

balanceInLamports = balance.fromLamports(decimals)